hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/AbfsHttpConstants.java

```diff
@@ -121,8 +121,34 @@ public final class AbfsHttpConstants {
   public static final char CHAR_EQUALS = '=';
   public static final char CHAR_STAR = '*';
   public static final char CHAR_PLUS = '+';
-  public static final String DECEMBER_2019_API_VERSION = "2019-12-12";
```

Comment:
this is a `@InterfaceAudience.Public` class, you can't remove this. 
What you can do here and below is
* declare the value underneath the new enum and set it from the  appropriate enum
* tag as deprecated

Reply:
Added it back as deprecated.
Thanks for pointing this out

Reply:
Taken